.circleci/config.yml

```diff
@@ -3,9 +3,9 @@ version: 2.1
 matrix_goversions: &matrix_goversions
   matrix:
     parameters:
-      goversion: ["18", "19"]
+      goversion: ["18", "19", "20"]
 
-default_goversion: &default_goversion "18"
+default_goversion: &default_goversion "20"
```

Comment:
Should we update the go version in go.mod files for the library & examples to 1.20 too?

Reply:
hmm, this sounds like a familiar convo 😄 from what I recall, the version in the go mod file doesn't actually enforce anything, it's just like "what this file was last modified with". if that's that case, we just update it whenever someone adds a new or updates a new dependency while running a new Go version on their machine?